python/fusion_engine_client/applications/p1_print.py

```diff
@@ -123,6 +125,9 @@ def main():
         '--skip', type=int, default=0,
         help="Skip the first N messages in the log. If --message-type is specified, only count messages matching the "
              "specified type(s).")
+    parser.add_argument(
+        '--source-identifier', '--source-id', nargs='*', type=int, default=0,
+        help="Print the Fusion Engine messages characterized by the listed source identifier(s).")
```

Comment:
"characterized by" to "with" perhaps?

Reply:
Does this work for a list? What format should the user enter for a list?

Reply:
I suggest using `CSVAction` for arguments that can be specified more than once. See the use in `analyzer.py`.